wpears/financial-well-being
# Set up Capital Framework project

Sets up a basic Capital Framework project.

To test:

1. Start a server with `python -m SimpleHTTPServer`
2. In a new terminal tab, run `gulp clean && gulp build`
3. Run `gulp watch`
4. A demo CF page should pop up in your browser
